Lib/urllib/parse.py

```diff
@@ -366,9 +366,22 @@ def _fix_result_transcoding():
 def urlparse(url, scheme='', allow_fragments=True):
     """Parse a URL into 6 components:
     <scheme>://<netloc>/<path>;<params>?<query>#<fragment>
-    Return a 6-tuple: (scheme, netloc, path, params, query, fragment).
-    Note that we don't break the components up in smaller bits
-    (e.g. netloc is a single string) and we don't expand % escapes."""
+
+    The result is a ParseResult or ParseResultBytes object,
+    depending on the type of the URL parameter.
```

Comment:
The parameter is named "url" rather than "URL", let's keep the correct spelling.
```suggestion
    depending on the type of the url parameter.
```